runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetOverlay.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2019, Abiyaz C <chowdh2@cooper.edu>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package net.runelite.client.plugins.driftnet;
+
+import net.runelite.api.*;
+import net.runelite.api.geometry.SimplePolygon;
+import net.runelite.client.ui.overlay.Overlay;
+import javax.inject.Inject;
+import java.awt.*;
+import java.util.Map;
+import net.runelite.client.ui.overlay.OverlayLayer;
+import net.runelite.client.ui.overlay.OverlayPosition;
+
+public class DriftNetOverlay extends Overlay
+{
+	private final DriftNetConfig config;
+	private final DriftNetPlugin plugin;
+
+	@Inject
+	DriftNetOverlay(DriftNetConfig config, DriftNetPlugin plugin)
+	{
+		this.config = config;
+		this.plugin = plugin;
+		setPosition(OverlayPosition.DYNAMIC);
+		setLayer(OverlayLayer.ABOVE_SCENE);
+	}
+
+
+	@Override
+	public Dimension render(Graphics2D graphics)
+	{
+		if (config.netOverlay())
+		{
+			renderNet(graphics);
+		}
+		if (config.highlightFish())
+		{
+			renderFishes(graphics);
+		}
+		return null;
+	}
+
+	static Color getColor(DriftNet net)
+	{
+		Color color = null;
+		if (net.getDriftNet() != null)
+		{
+			switch (net.getNetStatus())
+			{
+				case FULL:
+					color = Color.RED;
+					break;
+				case SET:
+					color = Color.GREEN;
+					break;
+				case UNSET:
+					color = Color.YELLOW;
+					break;
+				default:
+					break;
+			}
+		}
+		return color;
+	}
```

Comment:
```suggestion
	static Color getColor(DriftNet net)
	{
		if (net.getDriftNet() == null)
		{
			return null;
		}
		switch (net.getNetStatus())
		{
			case FULL:
				return Color.RED;
			case SET:
				return Color.GREEN;
			case UNSET:
				return Color.YELLOW;
			default:
				return null;
		}
	}
```
Not sure how GitHub handles suggestions on multiline comments

Reply:
It may be worth storing the colour in the DriftNetStatus Enum, but I'm not sure.